index.html

```diff
@@ -11,7 +11,7 @@
   <body>
     <header>
       <div class="logo-container">
-        <h1 class="logo">My Logo</h1>
+        <h1 href="index.html" class="logo">My Logo</h1>
```

Comment:
- [ ] I think you forgot to change the `h1` tag to an `a`  🤓. Please ensure to fix that because I believe this is not a valid way to add a link. :ok_hand: 
```suggestion
<a href="index.html" class="logo">My Logo</a>
```